src/components/Clients/ClientListTableRow.jsx

```diff
@@ -0,0 +1,146 @@
+// React Imports
+import React, { useState, useContext } from 'react';
+// React Router Imports
+import { Link } from 'react-router-dom';
+// Material UI Imports
+import { styled, useTheme } from '@mui/material/styles';
+import Checkbox from '@mui/material/Checkbox';
+import IconButton from '@mui/material/IconButton';
+import TableCell, { tableCellClasses } from '@mui/material/TableCell';
+import TableRow from '@mui/material/TableRow';
+import DeleteOutlineOutlinedIcon from '@mui/icons-material/DeleteOutlineOutlined';
+import PushPinIcon from '@mui/icons-material/PushPin';
+import PushPinOutlinedIcon from '@mui/icons-material/PushPinOutlined';
+// Utility Imports
+import { runNotification } from '../../utils';
+// Context Imports
+import { SelectUserContext, UserListContext } from '../../contexts';
+
+// determine what gets rendered in the table body
+const ClientListTableRow = ({ loadingActive, labelId, client, state, dispatch }) => {
+  const theme = useTheme();
+  const { setSelectedUser } = useContext(SelectUserContext);
+  const { removeUser } = useContext(UserListContext);
+  const [pinned, setPinned] = useState(false);
+  const [selected, setSelected] = useState(false);
+
+  // determine what gets rendered in the date modified cell of table
+  const determineDateModifiedCell = () => {
```

Comment:
Think we can have this function removed since it would seem loadingActive is no longer in use in favor of having all data loaded in userList (including last active date) before passing it into PASS.

Reply:
Updated